README.md

```diff
@@ -17,4 +17,42 @@ Run `make build`.
 ## Testing
 
 Run `make test` to run all unit and integration tests. To run the integration test, set the Amazon Redshift database DSN
-to the environment variables as an `REDSHIFT_DSN`.
\ No newline at end of file
+to the environment variables as an `REDSHIFT_DSN`.
+
+## Source
+
+The Redshift Source Connector allows you to move data from a Redshift table with the specified `dsn` and `table`
+configuration parameters. Upon starting, the Source takes a snapshot of a given table in the database, then switches
+into CDC mode. In CDC mode, the connector will only detect new rows.
+
+### Snapshot Capture
+
+The first time the connector starts, the snapshot mode is enabled and the last value of the `orderingColumn` is stored
+to the position, to know the boundary of this mode. The connector reads all rows of a table in batches, using a
+cursor-based pagination, limiting the rows by `batchSize` and ordering by `orderingColumn`. The connector stores the
+last processed element value of an `orderingColumn` in a position, so the snapshot process can be paused and resumed
+without losing data. Once all rows in that initial snapshot are read the connector switches into CDC mode.
+
+This behavior is enabled by default, but can be turned off by adding `"snapshot": false` to the Source configuration.
+
+### Change Data Capture
+
+This mode moves only rows added after the first start of the connector in batches, using a cursor-based pagination,
```

Comment:
Perhaps you meant the [keyset pagination](https://use-the-index-luke.com/no-offset)?

Reply:
Yeah, maybe it's the better name 👍